math_science_reqs/24254.md

```diff
@@ -0,0 +1,25 @@
+# 21-254: Linear Algebra and Vector Calculus for Engineers
+
+| Category | Difficulty |
+| :------: | :--------: |
+|   HW     |     4      |
+|  Exams   |     4      |
+
+Linear Algebra and Vector Calculus for Engineers is a fast-moving class that covers a broad range of topics. It essentially packs the content of two courses into a single semester: the first half is linear algebra, such as computing matrices and eigenvalues (which is also a topic covered in 18202), then vector calculus (which is similar to the second half of 21259). Due to its breadth, it does cover material at much less depth, which you may or may not prefer. The course definitely seems to be oriented more for engineering, skipping over proofs and justifications and focusing more on calculations and using Matlab. 
```

Comment:
Do you mind linking the classes here to their docs? 18202 and 21259?

Reply:
All done, thanks for the bug catches!